index.html

```diff
@@ -12,24 +12,23 @@
     <title>Danicas Website</title>
   </head>
   <body>
-    <header id="top">
-      <h1>Danica Dombek</h1>
-      <nav>
-        <a href="#biographie" class="navi-buttons">Biographie</a>
-        <a href="#kontakt" class="navi-buttons">Kontakt</a>
+    <header id="top" class="header">
+      <h1 class="header__h1">Danica Dombek</h1>
+      <nav class="header__nav">
+        <a href="#biographie" class="header__navi-buttons">Biographie</a>
```

Comment:
Hier würd man eher nav statt Navi schreiben. Eventuell würde auch header__button reichen

Reply:
I'd name it `header__nav-link` or `header__link`:
```suggestion
        <a href="#biographie" class="header__nav-link">Biographie</a>
```